packages/cadl-python/src/emitter.ts

```diff
@@ -77,7 +78,8 @@ const EmitterOptionsSchema: JSONSchemaType<EmitterOptions> = {
         "basic-setup-py": { type: "boolean", nullable: true },
         "package-version": { type: "string", nullable: true },
         "package-name": { type: "string", nullable: true },
-        "output-path": { type: "string", nullable: true },
+        "output-dir": { type: "string", nullable: true },
```

Comment:
`--output-path` is deprecated in latest compiler and we had better use new name: https://github.com/microsoft/cadl/blob/main/packages/compiler/CHANGELOG.md#0370